### PR TITLE
Add management of systemd-oomd

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
Add parameters to manage systemd-oomd and its configuration.

### CODE REVIEW

**Comment by jcpunk** [comment]
I have no idea why those tests are failing...

File: manifests/oomd.pp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# @api private
+#
+# @summary This class provides an abstract way to setup systemd-oomd
+#
+# @param oomd_ensure The state that the ``oomd`` service should be in
+# @param oomd_settings Configuration elements for systemd-oomd
+class systemd::oomd (
+  Enum['stopped','running'] $oomd_ensure = $systemd::oomd_ensure,
+  Systemd::OomdSettings     $oomd_settings = systemd::oomd_settings,
```
Comment (smortex) [inline manifests/oomd.pp:9]: Typo:

```suggestion
  Systemd::OomdSettings     $oomd_settings = $systemd::oomd_settings,
```

**Comment by jcpunk** [comment]
That puppet7/ruby 2.7 test looks like it did something wonky on github?

**Comment by smortex** [comment]
The failure seems to happen on master too, when gathering ArchLinux facts (removing Arch from metadata.json fix CI on my machine):

https://github.com/voxpupuli/rspec-puppet-facts/blob/0df5a5d82b48f7bc87016e46b247b69a6f5c3302/lib/rspec-puppet-facts.rb#L160

@bastelfreak maybe this will ring a bell?

**Review by bastelfreak — APPROVED** [review]

**Comment by jcpunk** [comment]
Any chance for a merge?

**Pull request merged**